core/workspace_svg.js

```diff
@@ -167,6 +168,13 @@ Blockly.WorkspaceSvg = function(options,
    * @type {boolean}
    */
   this.keyboardAccessibilityMode = false;
+
+  /**
+   * The list of top-level bounding boxes on the workspace.
+   * @type {!Array.<!Blockly.IBoundingBox>}
```

Comment:
prefer `Array.<` => `Array<`
We are using both, though mostly using the dot syntax. With the dot is deprecated syntax, so it would be nice to have new things use the updated syntax and if it doesn't already exists, maybe can file a bug to update the rest.

Same comment for L2551

Reply:
Generally the style guide recommends to use the style of the project over what's specified in the style guide. Since Blockly was written at a time where the style guide recommended the dot syntax, `Array.<` is used in Blockly. With the introduction of blockly-samples we decided we would adopt the style guide as is there.

I see a few instances of `Array<` in Blockly. Will fix those.